frontend/app/src/store/defi/getters.ts

```diff
@@ -1412,7 +1420,7 @@ export const getters: Getters<DefiState, DefiGetters, RotkehlchenState, any> = {
           quoteAsset: trade.quoteAsset,
           rate: trade.rate,
           tradeType: 'buy',
-          link: '',
+          link: `${txUrl}${trade.txHash}`,
```

Comment:
One thing I don't remember here is if the setting automatically appends a slash at the end or not.

if we allow a URL without a slash at the end then we will have to take care of it here. otherwise, it will not work for all cases.

Reply:
In Hashlink.vue you sent me look via discord there was this kind of logic. Mayhaps this means is indeed not a gaurantee?